### PR TITLE
Add documents to a collection

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/C1qupIgZ/22-create-a-load-testing-scenario-to-include-documents-in-a-collection

- Creates a document collection 
- Searches for documents created from the Whitehall Publishing Gatling load test scenario.
- Adds these to the collection
- Tags to taxonomy
- Force publishes

Signon and tagging steps have been DRY'd up as part of this PR.

### CODE REVIEW

File: test-plans/Signon.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      .check(
+        regex("Sign in to GOV.UK").exists,
+        css("input[name=authenticity_token]", "value").saveAs("signonAuthToken")
+      )
+  )
+  .exec(
+    http("Authenticate in signon")
+      .post(signonUrl)
+      .formParam("authenticity_token", """${signonAuthToken}""")
+      .formParam("user[email]", sys.env.get("USERNAME").get)
```
Comment (barrucadu) [inline test-plans/Signon.scala]: Is there a reason these are environment variables while everything else is a property?

Comment (steventux) [reply test-plans/Signon.scala]: I don't suppose there is really, possibly due to how I was running the scenario: setting the creds in a distinct step, then exporting the properties as I ran the scenario. There's no reason why they can't be properties though. I'll update for consistency.

Comment (steventux) [reply test-plans/Signon.scala]: @barrucadu ah I remember now, yes there was a reason, spaces in JAVA_OPTS properties are problematic, at least for Gatling.
It looks like the Gatling startup script doesn't handle these correctly.
For example the property `-Dpassword="my information is secret"` results in `java.lang.ClassNotFoundException: information`.

Comment (barrucadu) [reply test-plans/Signon.scala]: Ah, shell scripts...

**Pull request merged**